app/assets/javascripts/basetypes/Code.js

```diff
@@ -1,9 +1,37 @@
 const mongoose = require('mongoose');
+const cql = require('cql-execution');
 
 function Code(key, options) {
   mongoose.SchemaType.call(this, key, options, 'Code');
 }
 Code.prototype = Object.create(mongoose.SchemaType.prototype);
 
+function Cast(code) {
```

Comment:
Can this function be called in basetypes/Any.js in the RecursiveCast function where we cast to code?

Reply:
I think this function might need to be written more like the cast functions on all of the other basetypes (Quantity, Interval, DateTime). eg: `Quantity.prototype.cast = (quantity) => {....`

Reply:
did that originally, and then eslint complained about an anonymous function. makes sense to change it.